.github/dependabot.yml

```diff
@@ -18,9 +18,6 @@ updates:
     directory: "/"
     schedule:
       interval: daily
-    assignees:
```

Comment:
Instead of removing the `assignees` entry you can add me (`waseem826`) or any other frontend dev here.